Amazon.Ion.ObjectMapper.Test/TestObjects.cs

```diff
@@ -286,4 +287,30 @@ public override string ToString()
             return "<Teacher>{ firstName: " + firstName + ", lastName: " + lastName + ", department: " + department + ", birthDate: " + birthDate + " }";
         }
     }
+    
+    public class UpperCaseStringIonSerializer : IonSerializer<string>
+    {
+        public string Deserialize(IIonReader reader)
+        {
+            return reader.StringValue().ToUpper();
+        }
+
+        public void Serialize(IIonWriter writer, string item)
+        {
+            writer.WriteString(item.ToUpper());
+        }
+    }
+    
+    public class NegativeIntIonSerializer : IonSerializer<int>
+    {
+        public int Deserialize(IIonReader reader)
+        {
+            return -reader.IntValue();
+        }
+
+        public void Serialize(IIonWriter writer, int item)
+        {
+            writer.WriteInt(-item);
+        }
+    }
 }
```

Comment:
Please add tests for custom serializers for other types.

Reply:
Added tests for each primitive type custom serializer.